packages/chrome-devtools/src/App.tsx

```diff
@@ -12,6 +12,7 @@ const { Content } = Layout;
 const App = () => {
   const [module, setModule] = useState({});
 
+  console.log('module', module);
```

Comment:
since there is a console.log in the package should this PR be marked as WIP?

Reply:
I found an existing problem, which has been repaired and should be able to continue the review status